packages/dashboard/package.json

```diff
@@ -8,13 +8,13 @@
     "lib/version.js"
   ],
   "scripts": {
-    "build": "node ../../scripts/just.js build",
-    "just": "node ../../scripts/just.js",
-    "clean": "node ../../scripts/just.js clean",
-    "start": "node ../../scripts/just.js dev",
+    "build": "npx just-scripts build",
+    "just": "npx just-scripts",
+    "clean": "npx just-scripts clean",
+    "start": "npx just-scripts dev",
     "test": "react-scripts-ts test --env=jsdom",
-    "start-test": "node ../../scripts/just.js jest-watch",
-    "update-snapshots": "node ../../scripts/just.js jest -u"
+    "start-test": "npx just-scripts jest-watch",
+    "update-snapshots": "npx just-scripts jest -u"
```

Comment:
The build was complaining about not finding `just.ts` and I saw other packages had changed to do this instead.